src/Kunstmaan/TranslatorBundle/Resources/config/services.yml

```diff
@@ -52,14 +52,14 @@ services:
 
     kunstmaan_translator.service.importer.importer:
         class: 'Kunstmaan\TranslatorBundle\Service\Command\Importer\Importer'
-        calls:
-            - [setTranslationGroupManager, ['@kunstmaan_translator.service.group_manager']] # default, maken via config var
+        arguments:
+            $translationGroupManager: '@kunstmaan_translator.service.group_manager'
         public: true
 
     kunstmaan_translator.service.group_manager:
         class: 'Kunstmaan\TranslatorBundle\Service\TranslationGroupManager'
-        calls:
-            - [setTranslationRepository, ['@kunstmaan_translator.repository.translation']]
+        arguments:
+            $translationRepository: '@kunstmaan_translator.repository.translation'
```

Comment:
We don't use autowiring in the bundles, so I would suggest to remove the parameter name here as this might be forgotten when renaming the parameter and the service creation will fail